timesketch/views/auth.py

```diff
@@ -52,6 +62,10 @@ def login():
         Redirect if authentication is successful or template with context
         otherwise.
     """
+    # Google OpenID Connect authentication.
+    if current_app.config.get(u'GOOGLE_OIDC_ENABLED', False):
```

Comment:
`from __future__ import unicode_literals`?

Reply:
This is going into a bigger refactor later. Will be in a separate PR.

Reply:
SG